documentation/antora.yml

```diff
@@ -29,6 +29,7 @@ asciidoc:
     link-mongodb-connector: 'connectors/mongodb.adoc'
     link-mongodb-event-flattening: 'configuration/mongodb-event-flattening.adoc' 
     link-mysql-connector: 'connectors/mysql.adoc'
+    link-mysql-connector-properties: 'connectors/mysql.adoc'
```

Comment:
@TovaCohen Why is it we needed this one, given it's exactly the same as the one above?

Reply:
Ok, I've logged https://issues.redhat.com/browse/DBZ-2223 for potentially removing this again. For now, we're merging this PR as is in order to move forward and it shouldn't make a difference in terms of the actual rendered page.

Reply:
I thought I needed both because the doc is in separate files. But I was wrong. 
I can remove this attribute, but I also need to update the file that refers to it. I'll create a separate commit and PR for that now.

Reply:
Follow up PR: https://github.com/debezium/debezium/pull/1621